tests/foreman/cli/test_remoteexecution.py

```diff
@@ -33,56 +34,47 @@
 from robottelo.cli.recurring_logic import RecurringLogic
 from robottelo.cli.task import Task
 from robottelo.config import settings
-from robottelo.constants import DISTRO_DEFAULT
-from robottelo.constants import DISTRO_RHEL7
-from robottelo.constants import DISTRO_SLES11
-from robottelo.constants import DISTRO_SLES12
 from robottelo.constants.repos import FAKE_0_YUM_REPO
 from robottelo.helpers import add_remote_execution_ssh_key
-from robottelo.vm import VirtualMachine
+from robottelo.hosts import ContentHost
 
 TEMPLATE_FILE = 'template_file.txt'
-TEMPLATE_FILE_EMPTY = 'template_file_empty.txt'
-distros = [DISTRO_DEFAULT]
 
 
 @pytest.fixture(scope="module")
 def fixture_org():
     org = entities.Organization().create()
     ssh.command(f'''echo 'echo Enforcing' > {TEMPLATE_FILE}''')
```

Comment:
is there any specific reason for running this command here? as this can be done at the test level as well and if this can be removed, so make use of the `module_org` api fixture to create org.

Reply:
historical reasons I suppose, moved to test level